### PR TITLE
ID: add bahasa translation for quickstart/beginner.ipynb

### PR DESCRIPTION
**Pull request by febriaroo**
Current reviewer:
- @LauraVieloma
- @rendrat

### CODE REVIEW

**Comment by lamberta** [comment]
Thanks! This is awesome.

I've applied some formatting fixes, but some things to keep in mind for future translations:

* Best to download the .ipynb file from Colab to keep the JSON formatted in a readable way.
* Make sure to update the button links to point to your notebooks in GitHub (replace `en/` with `id/`).
* Add the community translation disclaimer to the top of the file.

See the [docs contributor guide](https://www.tensorflow.org/community/contribute/docs#interactive_notebooks) and [notebook template](https://github.com/tensorflow/docs/blob/master/tools/templates/notebook.ipynb) for additional style rules.

Any ideas about reviewers? :) I'll send out a message to the [docs@tensorflow.org list](https://groups.google.com/a/tensorflow.org/forum/#!forum/docs) and see if we can start building a community. We should create a `site/id/REVIEWERS` file to notify people of new translations (for example, here's the [site/vi/REVIEWERS](https://github.com/tensorflow/docs/blob/master/site/vi/REVIEWERS) file).

**Comment by febriaroo** [comment]
Hi @lamberta! 
Thank you, I will keep that in mind :)

I'm not sure about the reviewer, but I have someone in mind that can help to review. How you want me to connect with them?

**Comment by lamberta** [comment]
Hi @febriaroo,
As far as reviewers, anyone with a GitHub account will do :)
I've added a `REVIEWERS` file to your pull request. Feel free to add your GitHub username to the bottom to get notified on future ID translations (other people can add themselves when this PR is merged). In the meantime, you can add their name to a GitHub comment to send them a notification. I'll leave this PR open for a bit to see if anyone jumps in, otherwise, I can merge it to help bootstrap the community. But, generally, we really want the community to be involved with the reviews since, well, my Bahasa Indonesian is not so good 😉

**Comment by isfaaghyth** [comment]
hi @febriaroo! This is one of the interesting things, and I want to be part of the contributors to this project with you

**Comment by lamberta** [comment]
@isfaaghyth (and others) The easiest way to stay involved is to add your username to the `site/id/REVIEWERS` file (when this PR lands).
Some other language groups have set up Slack/chat or use spreadsheets to communicate. Up to the language communities to organize how they'd like :)

**Comment by isfaaghyth** [comment]
@lamberta nice! let's make this PR landing then. 🚀

File: site/id/tutorials/quickstart/advanced.ipynb [hunk 1]
```diff
@@ -0,0 +1,424 @@
...
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "LKIURqSTZU4U",
+        "colab_type": "text"
+      },
+      "source": [
+        "Note: Komunitas TensorFlow kami telah menerjemahkan dokumen-dokumen ini. Tidak ada jaminan bahwa translasi ini akurat, dan translasi terbaru dari [Official Dokumentasi - Bahasa Inggris](https://www.tensorflow.org/?hl=en) karena komunitas translasi ini adalah usaha terbaik dari komunitas translasi.\n",
+        "Jika Anda memiliki saran untuk meningkatkan terjemahan ini, silakan kirim pull request ke [tensorflow/docs](https://github.com/tensorflow/docs) repositori GitHub.\n",
+        "Untuk menjadi sukarelawan untuk menulis atau memeriksa terjemahan komunitas, hubungi\n",
+        "[daftar docs@tensorflow.org](https://groups.google.com/a/tensorflow.org/forum/#!forum/docs)."
```
Comment (rendrat) [inline site/id/tutorials/quickstart/advanced.ipynb]: keep this to

> [docs@tensorflow.org list]

File: site/id/tutorials/quickstart/advanced.ipynb [hunk 2]
```diff
@@ -0,0 +1,424 @@
...
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "hiH7AC-NTniF"
+      },
+      "source": [
+        "Ini adalah file notebook [Google Colaboratory](https://colab.research.google.com/notebooks/welcome.ipynb). Program python akan dijalankan langsung dari browser — cara yang bagus untuk mempelajari dan menggunakan TensorFlow. Untuk mengikuti tutorial ini, jalankan notebook di Google Colab dengan mengklik tombol di bagian atas halaman ini.\n",
```
Comment (rendrat) [inline site/id/tutorials/quickstart/advanced.ipynb]: > cara yang bagus untuk mempelajari dan menggunakan TensorFlow

please edit to 

> ini adalah cara terbaik untuk belajar menggunakan TensorFlow.

File: site/id/tutorials/quickstart/advanced.ipynb [hunk 3]
```diff
@@ -0,0 +1,424 @@
...
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "JB6A1vcigsIe"
+      },
+      "source": [
+        "Pilih metrik untuk mengukur loss dan keakuratan model. Metrik ini mengakumulasi nilai di atas epochs dan kemudian mencetak hasil secara keseluruhan."
```
Comment (rendrat) [inline site/id/tutorials/quickstart/advanced.ipynb]: > nilai di atas epochs dan kemudian

please edit to 

> nilai yang lebih besar dari epochs kemudian

and

> mengukur loss dan keakuratan model

to

> mengukur kerugian dan keakuratan model

File: site/id/tutorials/quickstart/advanced.ipynb [hunk 4]
```diff
@@ -0,0 +1,424 @@
...
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "T4JfEh7kvx6m"
+      },
+      "source": [
+        "Penggolong gambar tersebut, sekarang dilatih untuk akurasi ~ 98% pada dataset ini. Untuk mempelajari lebih lanjut, baca [tutorial TensorFlow](https://www.tensorflow.org/tutorials/)."
```
Comment (rendrat) [inline site/id/tutorials/quickstart/advanced.ipynb]: > Penggolong gambar tersebut, sekarang dilatih untuk akurasi ~ 98% pada dataset ini.

Please edit to

> Pengelompokan gambar kini di latih untuk mendapatkan akurasi ~98% pada dataset ini.

File: site/id/tutorials/quickstart/beginner.ipynb [hunk 5]
```diff
@@ -0,0 +1,257 @@
...
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "DJELZj47ZL1o",
+        "colab_type": "text"
+      },
+      "source": [
+        "Note: Komunitas TensorFlow kami telah menerjemahkan dokumen-dokumen ini. Tidak ada jaminan bahwa translasi ini akurat, dan translasi terbaru dari [Official Dokumentasi - Bahasa Inggris](https://www.tensorflow.org/?hl=en) karena komunitas translasi ini adalah usaha terbaik dari komunitas translasi.\n",
```
Comment (rendrat) [inline site/id/tutorials/quickstart/beginner.ipynb]: > dan translasi terbaru dari

please edit to

> dan translasi terbaru dapat dilihat di

and

> karena komunitas translasi ini adalah usaha terbaik dari komunitas translasi.

to

> karena ini adalah usaha terbaik yang telah dilakukan dari komunitas kami.

File: site/id/tutorials/quickstart/beginner.ipynb [hunk 6]
```diff
@@ -0,0 +1,257 @@
...
+        "Jika Anda memiliki saran untuk meningkatkan terjemahan ini, silakan kirim pull request ke [tensorflow/docs](https://github.com/tensorflow/docs) repositori GitHub.\n",
+        "Untuk menjadi sukarelawan untuk menulis atau memeriksa terjemahan komunitas, hubungi\n",
+        "[daftar docs@tensorflow.org](https://groups.google.com/a/tensorflow.org/forum/#!forum/docs)."
```
Comment (rendrat) [inline site/id/tutorials/quickstart/beginner.ipynb]: keep this to

> [docs@tensorflow.org list]

File: site/id/tutorials/quickstart/beginner.ipynb [hunk 7]
```diff
@@ -0,0 +1,257 @@
...
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "hiH7AC-NTniF"
+      },
+      "source": [
+        "Ini adalah file notebook [Google Colaboratory](https://colab.research.google.com/notebooks/welcome.ipynb). Program python akan dijalankan langsung dari browser — cara yang bagus untuk mempelajari dan menggunakan TensorFlow. Untuk mengikuti tutorial ini, jalankan notebook di Google Colab dengan mengklik tombol di bagian atas halaman ini.\n",
```
Comment (rendrat) [inline site/id/tutorials/quickstart/beginner.ipynb]: > cara yang bagus untuk mempelajari dan menggunakan TensorFlow

please edit to

> ini adalah cara terbaik untuk belajar menggunakan TensorFlow.

File: site/id/tutorials/quickstart/beginner.ipynb [hunk 8]
```diff
@@ -0,0 +1,257 @@
...
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "T4JfEh7kvx6m"
+      },
+      "source": [
+        "Penggolong gambar tersebut, sekarang dilatih untuk akurasi ~ 98% pada dataset ini. Untuk mempelajari lebih lanjut, baca [tutorial TensorFlow](https://www.tensorflow.org/tutorials/)."
```
Comment (rendrat) [inline site/id/tutorials/quickstart/beginner.ipynb]: > Penggolong gambar tersebut, sekarang dilatih untuk akurasi ~ 98% pada dataset ini.

Please edit to

> Pengelompokan gambar kini di latih untuk mendapatkan akurasi ~98% pada dataset ini.

File: site/id/tutorials/quickstart/beginner.ipynb [hunk 9]
```diff
@@ -0,0 +1,257 @@
...
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "7NAbSZiaoJ4z"
+      },
+      "source": [
+        "Siapkan [dataset MNIST](http://yann.lecun.com/exdb/mnist/). Ubah sampel dari bilangan bulat menjadi angka floating-point (desimal):"
```
Comment (rendrat) [inline site/id/tutorials/quickstart/beginner.ipynb]: Please edit this 
> Ubah sampel dari bilangan bulat menjadi angka floating-point (desimal)

to 

> Ubah sampel dari bilangan bulat menjadi bilangan desimal

**Pull request merged**